src/app/core/search-home/search-home.component.ts

```diff
@@ -77,44 +70,43 @@ export class SearchHomeComponent implements OnInit, OnDestroy {
     }
   }
 
-  public defaultAttribute(document: SearchDocument): string {
-    return document && document.document && document.document.data ? Object.values(document.document.data)[0] : '';
+  public defaultAttribute(document: DocumentModel): string {
+    return document && document.data && document.data ? Object.values(document.data)[0] : '';
```

Comment:
Why do we have `document.data` twice here?

Reply:
It was just quick refactor so I forgot it here